### PR TITLE
Implement Rakam

### LINKED ISSUE #8442: Intent-to-Implement: Rakam
I would love to add Rakam as an amp-analytics plugin, the implementation is similar to Segment's PR and pretty straightforward.

A simple example that sends a page view:
```
<amp-analytics type="rakam">
<script type="application/json">
  {
    "vars": {
      "apiEndpoint": "<rakam api url>",
      "writeKey": "<key>",
    }
  }
...
```

### PR DESCRIPTION
**Pull request by buremba**
Fixes #8442

### CODE REVIEW

**Comment by buremba** [comment]
I signed it!

**Comment by dvoytenko** [comment]
@buremba Thanks for the PR!

/to @zhouyx for review.

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -1229,6 +1229,33 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
...
+        '&prop._referrer=${documentReferrer}' +
+        '&prop.title=${title}' +
+        '&prop.timezone=${timezone}' +
+        '&prop._time=${timestamp}' +
+        '&prop.resolution=${screenWidth} × ${screenHeight}',
+      'pageview': 'https://${apiEndpoint}/event/pixel${base}&collection=${pageViewName}',
+      'custom': 'https://${apiEndpoint}/event/pixel${base}&collection=${collection}',
+    },
+    'transport': {
+      'beacon': false,
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors.js]: in case this is a copy from other config. 
`beacon: true` is recommended if your server does support it.

Comment (buremba) [inline extensions/amp-analytics/0.1/vendors.js]: We also accept POST requests so I enabled that feature as well.

**Comment by buremba** [comment]
@zhouyx thanks for the quick review! I made the changes you requested.

File: examples/analytics-vendors.amp.html [hunk 2]
```diff
@@ -683,6 +710,7 @@
 </script>
 </amp-analytics>
 <!-- End Ibeat Analytics example -->
+>>>>>>> ampproject/master
```
Comment (zhouyx) [inline examples/analytics-vendors.amp.html]: please fix the merge conflict

File: extensions/amp-analytics/amp-analytics.md [hunk 3]
```diff
@@ -286,6 +286,12 @@ Type attribute value: `metrika`
 
 Adds support for Yandex Metrica.  Configuration details can be found at [Yandex Support](https://yandex.com/support/metrica/code/install-counter-amp.xml).
 
+### Rakam
```
Comment (zhouyx) [inline extensions/amp-analytics/amp-analytics.md]: Please insert the doc following alphabetic order

File: extensions/amp-analytics/0.1/vendors.js [hunk 4]
```diff
@@ -1290,7 +1290,32 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
...
+        '&prop.color_depth=${screenColorDepth}' +
+        '&prop._referrer=${documentReferrer}' +
+        '&prop.title=${title}' +
+        '&prop.timezone=${timezone}' +
+        '&prop._time=${timestamp}' +
+        '&prop.resolution=${screenWidth} × ${screenHeight}',
+      'pageview': 'https://${apiEndpoint}/event/pixel${base}&collection=${pageViewName}',
+      'custom': 'https://${apiEndpoint}/event/pixel${base}&collection=${collection}',
+    },
+    'transport': {
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors.js]: Oh! You don't need to specify `transport` in this case. They are by default all true. Please remove.

File: examples/analytics-vendors.amp.html (hunk 2 above)
Comment (buremba) [inline examples/analytics-vendors.amp.html]: Oops, sorry for not noticing it.

**Comment by zhouyx** [comment]
@buremba Thanks for the Pull Request. LGTM. Merged

**Comment by buremba** [comment]
Thanks @zhouyx !

**Pull request merged**